inventory-service/src/main/java/com/petclinic/inventoryservice/businesslayer/ProductInventoryService.java

```diff
@@ -0,0 +1,9 @@
+package com.petclinic.inventoryservice.businesslayer;
+
+import com.petclinic.inventoryservice.presentationlayer.ProductRequestDTO;
+import com.petclinic.inventoryservice.presentationlayer.ProductResponseDTO;
+import reactor.core.publisher.Mono;
+
+public interface ProductInventoryService {
+    Mono<ProductResponseDTO> addProductToInventory(Mono<ProductRequestDTO> productRequestDTOMono, String inventoryId);
```

Comment:
Any reason the inventoryId is not provided with the productRequestDto?

Reply:
Because it is passed in the URI